templates/generate_template.sh

```diff
@@ -96,10 +96,6 @@ sh "${EBROOTMOLGENISMINCOMPUTE}/molgenis_compute.sh" \
 -w "${EBROOTNGS_DNA}/create_in-house_ngs_projects_workflow.csv" \
 -rundir "${genScripts}/scripts" \
 --runid "${runID}" \
--b slurm \
---header "${EBROOTNGS_DNA}/templates/slurm/header_tnt.ftl" \
---footer "${EBROOTNGS_DNA}/templates/slurm/footer_tnt.ftl" \
---submit "${EBROOTNGS_DNA}/templates/slurm/submit.ftl" \
```

Comment:
This is not working in combination with the NGS_Automated. The scripts were not done and the automated pipeline wanted already to navigate to the jobs folder, which is not there yet.